manager.cpp

```diff
@@ -272,8 +277,104 @@ void Manager::updateObjects(
     }
 }
 
-void Manager::notify(std::map<sdbusplus::message::object_path, Object> objs)
+void Manager::clearPersistency(const vector<string>& paths)
+{
+    string systemPath =
+        "/var/lib/phosphor-inventory-manager/xyz/openbmc_project/inventory";
```

Comment:
if elsewhere in the codebase gets the /var/lib/.. prefix from some other constant, should probably do that here too.